### PR TITLE
 Quiz result page

### PR DESCRIPTION
**Pull request by mehulsatardekar**
**Added quiz result page.**

 **preview**: https://deploy-preview-5--quizgrad.netlify.app/views/quiz-result

quiz-result

### CODE REVIEW

File: views/quiz-result.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            </ul>
+        </div>
+    
+        <div class="flex flex-space-evenly flex-align-item-center ">
+            <ul class=" text-deocration-none flex flex-wrap gap-col-2">
+               
+                <a href="#" class="text-deocration-none" target="_blank" rel="noreferrer">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons">help_outline</span>
+                        <span class="">help</span>
```
Comment (Shivani-1524) [inline views/quiz-result.html]: remove the unused span class

File: views/quiz-result.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                <div class="flex flex-column gap-2 mb-3 mt-3">
+                    <div class="flex flex-between">
+                        <span class="label-text label-text-primary">Question 1</span>
+                        <span class="label-text label-text-success">Scored: 10</span>
+                     </div>
+                    <span class="font-bold font-weight-800 font-sm">Where is Santorini located?</span>
+                    <span class="btn-success btn btn-py-1 flex flex-justify-center disable-btn disable">
+                        <span class="icon-default-pl"> South part of Greece, in the Aegean Sea, </span>
+                    </span>
+                    <span class="btn-outline-primary btn btn-py-1 flex flex-justify-center disable-text disable">
```
Comment (Shivani-1524) [inline views/quiz-result.html:75]: please try to increase the contrast between text and backgorund color

File: views/quiz-result.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                        <span class="">Sign-out</span>
+                    </li>
+                </a>
+            </ul>
+        </div>
+        
+    </nav>  
+
+
+    <main class="container">
```
Comment (Shivani-1524) [inline views/quiz-result.html:52]: nice use of semantics btw

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]

File: views/quiz-result.html (hunk 1 above)
Comment (mehulsatardekar) [reply views/quiz-result.html:31]: > remove the unused span class

@Shivani-1524 actually I am using google icons so to use them basically have to follow these structures.

**Review by gauravsinhaweb — APPROVED** [review]
Options are basically buttons, so you can use cursor pointer for that.  Good to merge after changes. Nice work Mehul!

File: views/quiz-result.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                        <span class="icon-default-pl"> It is located in bay of naples, italy. </span>
+                    </span>
+                    <span class="btn-outline-primary btn btn-py-1 flex flex-justify-center disable-text disable ">
+                        <span class="icon-default-pl"> It is somewhere in scotland. </span>
+                    </span>
+                </div> 
+            </section>
+        </section>
+        
+
```
Comment (Adarsh-Patel-Dev) [inline views/quiz-result.html]: just remove extra spaces here 
Rest code is clean.
It can be merged.

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Comment by mehulsatardekar** [comment]
> Options are basically buttons, so you can use cursor pointer for that. Good to merge after changes. Nice work Mehul!

@gauravsinhaweb thx for the suggestion, since its a result page, so making these options clickable won't make any difference for me

**Review by Shivani-1524 — APPROVED** [review]

**Pull request merged**